### PR TITLE
New pages & modifications

### PR DESCRIPTION
**Pull request by jessiwright**
Batch update, new pages & updates

### CODE REVIEW

File: docs/organizations/platform-standards.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Platform Standards help provide consistency, fairness, and, above all, the best results possible for all participants on the platform. Deviations from Platform Standards are not recommended and must be disclosed in a program’s policy if they exist.
+
+ Issue | Standard | Example
+----------- | ------------- | -------------
+Self Sign-Up / CVSS Privilege Requirement (PR) | If self-sign-up is possible, set the **Privileges Required** metric to None. | If a hacker finds a bug in an application that requires them to sign up and create their own credentials and does not require elevated privileges beyond that, then the CVSS calculation should include **PR:N**.
+Mass Leakage of Sensitive PII | When awarding a bounty, always consider this as a Critical severity issue. In serious scenarios where sensitive information is disclosed, CVSS may not always capture the full impact. | If a hacker finds a vulnerability that results in a mass leak of PII beyond the basic name/email (examples: phone numbers, addresses, government IDs, hashed passwords, PHI) the report should be rated at the **Critical** severity.
```
[the remark this answers is not in the document]
Comment (jessiwright) [reply docs/organizations/platform-standards.md:16]: Fixed

[the remark this answers is not in the document]
Comment (jessiwright) [reply docs/organizations/platform-standards.md:16]: Fixed

File: docs/hackers/tax-forms.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ To sign your tax form:
 
 6. If you make a mistake, you have the ability to start over without involving Support. On the **Settings > Payments page**, you will have an opportunity to abandon an in-progress tax form, and you will receive confirmation through your email that this has occurred.
 
-You can refer to these links for instructions on how to fill out your respective tax form:
+You can refer to these links for instructions on how to fill out your respective tax form. Please note for American citizens living internationally that the instructions will not tell you how to fill out your W-9. Please use commas in the "city/state/zip" box to format your international address as: "City, Postal Code, Country.”
```
[the remark this answers is not in the document]
Comment (jessiwright) [reply docs/hackers/tax-forms.md:26]: Any idea why it sometimes pastes them as smart quotes and sometimes as the ascii? This was one copy/paste function.

File: docs/hackers/tax-forms.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ To sign your tax form:
 
 6. If you make a mistake, you have the ability to start over without involving Support. On the **Settings > Payments page**, you will have an opportunity to abandon an in-progress tax form, and you will receive confirmation through your email that this has occurred.
 
-You can refer to these links for instructions on how to fill out your respective tax form:
+You can refer to these links for instructions on how to fill out your respective tax form. Please note for American citizens living internationally that the instructions will not tell you how to fill out your W-9. Please use commas in the "city/state/zip" box to format your international address as: "City, Postal Code, Country."
```
[the remark this answers is not in the document]
Comment (jessiwright) [reply docs/hackers/tax-forms.md:26]: no, periods go inside quotations

File: docs/organizations/retesting.md [hunk 4]
```diff
@@ -34,9 +34,9 @@ If you choose to:
...
+**Reject** the retest | The hacker says the vulnerability is fixed. | You’ll need to provide a summary to the hacker explaining why you’ve rejected the retest. You can choose to request another retest for the report, by going back to step 1. <br><br>The status of the report will be changed to it's previous state..
 **Approve** the retest | The hacker says the vulnerability is not fixed. | The report will move back to *Triaged* and will stay open for the team to implement a fix. The hacker will be awarded a bounty.
-**Reject** the retest | The hacker says the vulnerability is not fixed. | You’ll need to provide a summary to the hacker explaining why you’ve rejected the retest. You can choose to request another retest for the report, by going back to step 1. <br><br>The status of the report will be changed to *Triaged*.
+**Reject** the retest | The hacker says the vulnerability is not fixed. | You’ll need to provide a summary to the hacker explaining why you’ve rejected the retest. You can choose to request another retest for the report, by going back to step 1. <br><br>The status of the report will be changed to it's previous state.
```
[the remark this answers is not in the document]
Comment (jessiwright) [reply docs/organizations/retesting.md]: sorry, pasted it

**Review by martijnrusschen — APPROVED** [review]

**Pull request merged**